src/utils/xcmApi/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  createSubstrateAPI,
+  CollateralUnit,
+  DefaultTransactionAPI
+} from '@interlay/interbtc-api';
+import {
+  Currency,
+  MonetaryAmount
+} from '@interlay/monetary-js';
+import { web3FromAddress } from '@polkadot/extension-dapp';
+import { AddressOrPair } from '@polkadot/api/types';
+
+import {
+  RELAYCHAIN_URL,
+  RELAYCHAIN_ID
+} from '../../constants';
+
+const xcmTransfer = async (
+  // Pass this in explicitly to avoid creating
+  // a dependency on the interBCT api instance
+  originatingAccount: AddressOrPair,
+  destinationAddress: string,
+  transferAmount: MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>
+): Promise<void> => {
+  // Create api and transaction api instances on the parachain
+  const api = await createSubstrateAPI(RELAYCHAIN_URL);
+  const transactionApi = new DefaultTransactionAPI(api, originatingAccount);
+
+  // Signer needs to be set explicitly for the new api
+  const { signer } = await web3FromAddress(originatingAccount.toString());
+  api.setSigner(signer);
+
+  const dest = api.createType('XcmVersionedMultiLocation', {
+    v1: api.createType('XcmV1MultiLocation', {
+      parents: 0,
+      interior: api.createType('XcmV1MultilocationJunctions', {
+        x1: api.createType('XcmV1Junction', {
```

Comment:
Rather than nesting type creation, it's more readable to do it separately. E.g.:
```JS
const x1 = api.createType('XcmV1Junction', { parachain: RELAYCHAIN_ID });
const interior = api.createType('XcmV1MultilocationJunctions', { x1: x1});
const v1 = api.createType('XcmV1MultiLocation', { parents: 0, interior: interior });
...etc

```

Reply:
Same for the `beneficiary` and `assets`

Reply:
Agreed :) these variables are now being returned from functions (I found this easier to follow than having them all declared in the parent function and then namespaced).